### PR TITLE
Create Revert Button.

### PR DESCRIPTION
**Pull request by e0ipso**
Task: #143

* [x] Ready for review
* [x] Ready for merge

### CODE REVIEW

File: modules/contenta_enhancements/src/Form/RevertForm.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  public function getFormId() {
+    return 'contenta_enhancements.revert.confirm';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    // Use the admin theme in the front-end.
+    $this->themeConfig->set('default', 'material_admin');
```
Comment (dawehner) [inline modules/contenta_enhancements/src/Form/RevertForm.php]: Are you missing uninstalling the frontend theme on purpose?

Comment (e0ipso) [reply modules/contenta_enhancements/src/Form/RevertForm.php]: Not really. We can probably uninstall it as well

Comment (e0ipso) [reply modules/contenta_enhancements/src/Form/RevertForm.php]: done.

**Comment by dawehner** [comment]
Sorry I would have merged earlier, but sadly my phone crashed twice on looking at the diff.

**Pull request merged**